platform/api/oci/instance.go

```diff
@@ -37,7 +37,8 @@ func (a *API) CreateInstance(name, userdata string) (*Machine, error) {
 
 	opts := baremetal.LaunchInstanceOptions{
 		Metadata: map[string]string{
-			"user_data": base64.StdEncoding.EncodeToString([]byte(userdata)),
+			"user_data":           base64.StdEncoding.EncodeToString([]byte(userdata)),
+			"ssh_authorized_keys": sshKey,
```

Comment:
Do we need to omit this field if `sshKey` is empty?

Reply:
Pushed a change to omit it when empty